vite.config.mjs

```diff
@@ -1,9 +1,10 @@
-import { defineConfig, UserConfig } from 'vite'
+import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 import tsconfigPaths from 'vite-tsconfig-paths'
 import tailwindcss from '@tailwindcss/vite'
 
-export default defineConfig((): UserConfig => {
+/** @type {import('vite').UserConfig} */
+export default defineConfig(() => {
```

Comment:
What is going on here? It's not obvious, just from reading the code.

Reply:
keep IntelliSense in mjs